src/Microsoft.DotNet.XHarness.CLI/iOS/iOSTestCommand.cs

```diff
@@ -19,10 +29,15 @@ internal class iOSTestCommand : TestCommand
 
         public iOSTestCommand() : base()
         {
-            Options = new OptionSet() {
+            Options = new OptionSet()
+            {
                 "usage: ios test [OPTIONS]",
                 "",
                 "Packaging command that will create a iOS/tvOS/watchOS or macOS application that can be used to run NUnit or XUnit-based test dlls",
+                { "xcode=", "Path where Xcode is installed", v => _arguments.XcodeRoot = v},

```

Comment:
@mandel-macaque some of these could share an interface between the package and test commands.

Reply:
Also, not sure, but I probably changed the name from xcode-root and mlaunch-path. We can revisit that as well. But should be similar to whatever Android has. Or maybe later we could just revisit everything once we are stabilized so that we are unified across the CLI

Reply:
agreed, lets land as it is, since the API is not closed yet and this is a preview, we are allowed to change it.